README.md

```diff
@@ -1,73 +1,87 @@
+## Built With
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+* ReactJS
+* Redux
+* Material UI
+* HTML5
+* CSS3
 
 
-To get a local copy up and running follow these simple example steps.
+## Getting Started
 
-### Prerequisites
+* Clone this repo <https://github.com/white3d/react-covid-metrics-app.git>
 
-### Setup
+    ```bash
+    git clone https://github.com/white3d/react-covid-metrics-app.git
+    ```
 
-### Install
+* Navigate to react-covid-metrics-app folder/directory
 
-### Usage
+    ```bash
+    cd react-covid-metrics-app
+    ```
 
-### Run tests
+* On the comandline, at the project's root, run ```npm install``` to install app dependencies
 
-### Deployment
+* Next, run ```npm start``` which will run the app in the development mode.
 
+* Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
+* ALTERNATIVELY
 
-## Authors
+  * Just run ```npm run build``` which will build the project and generate output files into the ```build``` directory.
 
-👤 **Author1**
+  * Go to ```build``` directory and manually open ```index.html``` to interact with the app
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-👤 **Author2**
+## Contributing
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+Contributions, issues, and feature requests are welcome!
 
-## 🤝 Contributing
+Feel free to check the [issues page](../../issues)
 
-Contributions, issues, and feature requests are welcome!
+  1. Fork the Project
+  2. Create your Feature Branch (`git checkout -b feature/newFeature`)
+  3. Commit your Changes (`git commit -m 'Add some newFeature'`)
+  4. Push to the Branch (`git push -u origin feature/newFeature`)
+  5. Open a Pull Request
 
-Feel free to check the [issues page](../../issues/).
+## Authors
 
-## Show your support
+👤 **Edward Odhiambo**
 
-Give a ⭐️ if you like this project!
+- GitHub: [@whit3d](https://github.com/white3d)
+- Twitter: [@odhiambo_ed](https://twitter.com/odhiambo_ed)
+- LinkedIn: [Edward Odhiambo](https://www.linkedin.com/in/edward-odhiambo-6a462a21b/)
 
-## Acknowledgments
+## Acknowledgements
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+* [ReactJS Organization](https://reactjs.org/) for the amazing [Documentation](https://reactjs.org/docs/getting-started.html) on React.
+* [Redux Organization](https://redux.js.org/) for the amazing [Redux Essentials Tutorial](https://redux.js.org/tutorials/essentials/part-1-overview-concepts) on React.
 
-## 📝 License
+## Show your support
 
```

Comment:
- [ ] As requested by the previous code reviewer, which is a [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md) for this project, kindly add "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).